gpkit/nomials/nomial_math.py

```diff
@@ -581,7 +583,11 @@ def _gen_unsubbed(self):
         if l_lt_r.units:
             try:
                 l_lt_r.convert_to('dimensionless')
+                l_lt_r.cs = l_lt_r.cs.magnitude
+                l_lt_r.units = None
                 r_lt_l.convert_to('dimensionless')
+                r_lt_l.cs = r_lt_l.cs.magnitude
+                r_lt_l.units = None
```

Comment:
since we do it 3 times in this file, maybe we should define

```python
def non_dimensionalize(posy):
    "Non-dimensionalize a posy (warning: mutates posy)"
    posy.convert_to('dimensionless')
    posy.cs = posy.cs.magnitude
    posy.units = None
```